commit 4cad13584cf048c3130ec70b2ecf6988e0b01adc
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Jun 20 12:20:01 2018 +0100

    brcmfmac: Re-enable firmware roaming support
    
    As of 4.18, a firmware that implements the update_connect_params
    method but doesn't claim to support roaming causes an error. We
    disabled firmware roaming in 4.4 [1] because it appeared to
    prevent disconnects, but let's try with the current firmware to see
    if things have improved.
    
    [1] https://github.com/raspberrypi/linux/commit/dd9188011786fb62a7960922f31e8e086fb2009b
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c
index b00f0565c9b7..f288963011b2 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c
@@ -70,7 +70,7 @@ static int brcmf_fcmode;
 module_param_named(fcmode, brcmf_fcmode, int, 0);
 MODULE_PARM_DESC(fcmode, "Mode of firmware signalled flow control");
 
-static int brcmf_roamoff = 1;
+static int brcmf_roamoff;
 module_param_named(roamoff, brcmf_roamoff, int, 0400);
 MODULE_PARM_DESC(roamoff, "Do not use internal roaming engine");
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1502_4cad13584cf048c3130ec70b2ecf6988e0b01adc.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1502_4cad13584cf048c3130ec70b2ecf6988e0b01adc.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
